assets/components/content-types/laboratory/laboratory.scss

```diff
@@ -1 +1,286 @@
 @charset 'utf-8';
+
+
+
+.table-like {
+  $table-cours-template-columns: minmax(200px, 3fr) minmax(auto, 1fr) minmax(200px, 3fr);
+  $table-grid-template-columns: 1fr $table-cours-template-columns 2fr 2fr 2fr minmax(200px, 3fr) 1fr;
+
+  --table-columns: #{$table-grid-template-columns};
+
+  overflow-x: scroll;
+  max-width: 100%;
+
+  h4 {
+    font-weight: $font-weight-bold;
+    font-size: $font-size-base;
+  }
+
+  // Lines
+  // Grid properties are overridden on browsers supporting "display: contents".
+  .line,
+  .first-line {
+    display: grid;
+    grid-template-columns: $table-grid-template-columns;
+    font-size: 0.9375 * $font-size-base;
+
+    // Cells
+    > div {
+      padding: $table-cell-padding;
+      border-bottom: $table-border-width solid $table-border-color;
+
+      &:first-child {
+        border-left: $table-border-width solid $table-border-color;
+      }
+
+      &:last-child {
+        border-right: $table-border-width solid $table-border-color;
+      }
+
+      + div {
+        border-left: $table-border-width solid $table-border-color;
+      }
+    }
+  }
+
+  .first-line {
+    color: $table-head-color;
+    font-weight: $font-weight-bold;
+    text-transform: uppercase;
+    font-size: 0.6875 * $font-size-base;
+
+    > div {
+      border-top: $table-border-width solid $table-border-color;
+      border-bottom-width: $table-border-width * 2;
+    }
+  }
+
+  .line-up {
+    position: sticky;
```

Comment:
couldn't see any effect of this sticky declaration. Is that intended?

Reply:
Yes the table head sticks on supporting browsers.